File: Sem1/DSA/stack.java
import java.util.Stack;
public class stack{
    public static void main(String[] args){
          Stack<Integer> s = new Stack<Integer>();
          System.out.println(s);
          s.push(1); 
          s.push(3);
          int s1 = s.peek(); // Top of the stack
          System.out.println(s1);
          s.pop();
          System.out.println(s);
          s.peek();
          System.out.println(s);
    }
}

File: Sem1/DSA/stack.c
#include<stdio.h>
int top = -1;
#define max 20
int arr[max];

void peek(){
    int i;
    if(top > -1){
    for(i = 0; i <= top; i++){
           printf("%4d",arr[i]);
    }
    printf("\n");
    }
    else{
        printf("The stack is empty...\n");
    }
}

void push(int item){
     if(top == max - 1){
        printf("Stack Overflow...\n");
     }
     else{
        top++;
        arr[top] = item;
        peek();
     }
}

void pop(){
    if(top == -1){
        printf("The stack is empty...\n");
    }
    else{
        top--;
        peek();
    }
}

int main(){
    pop();
    push(10);
    push(20);
    push(30);
    pop();
    peek();
    push(50);
    peek();
    push(40);
    push(60);
    pop();
    push(70);
}